### DISCUSSION TITLE
How do I delete a playlist?

### DISCUSSION THREAD

**Discussion by IO101010** [navidrome/navidrome#3261] [Q&A]
Navidrome detected my music, but for each folder that had an m3u file it has created a playlist entry. (It actually created two duplicate playlists for each m3u file).

I don't want any playlists in Navidrome so I deleted the m3u files and did a scan. They're still present, though. I tried logging out and back in but they're still there. 

I found a comment by @deluan in a GitHub issue suggesting to run the command: `sqlite3 ./data/navidrome.db 'delete from playlist'` but when I ran that I get the error: Error: stepping, attempt to write a readonly database (8)

I'm using the current docker image. Can anyone help?

**Comment by deluan** [comment]
You'll need to disable the [`AutoImportPlaylists` config option](https://www.navidrome.org/docs/usage/configuration-options/#:~:text=%2220s%22-,AutoImportPlaylists,-ND_AUTOIMPORTPLAYLISTS) or else Navidrome will keep importing any m3u file it finds in your library.

Once imported, they can only be deleted manually or using the _workaround_ I posted about using sqlite3. Seems that you are getting a permission error, the user you are logged on does not have write access to the `./data/navidrome.db` file. Try doing that as `root`?

**Reply by IO101010** [reply]
Ah, running sqlite3 command with sudo did the trick - thank you! I can work with that.

I've also added: ND_AUTOIMPORTPLAYLISTS: "false" to docker-compose.
